ui/admin/app/components/form/host-set/aws/index.js

```diff
@@ -0,0 +1,17 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class FormHostSetAwsComponent extends Component {
+  // =actions
+  @tracked showSyncIntervalSeconds = true;
+
+  @action
+  toggleSyncIntervalSeconds(model) {
+    this.showSyncIntervalSeconds = !this.showSyncIntervalSeconds;
+
+    if (!this.showSyncIntervalSeconds) {
```

Comment:
What is the type of this.showSyncIntervalSeconds? asking because in case is boolean we might need to check if exists differently.